src/threads/decrypt.js

```diff
@@ -3,7 +3,7 @@
 const fs = require('fs');
 const crypto = require('crypto');
 const utils = require('#utils');
-const defaultConfig = require('../resources/defaultConfig.json');
+const defaultConfig = require('../../resources/defaultConfig.json');
```

Comment:
BTW why are we still using `defaultConfig.json` file instead of `config` module here and in the `encrypt.js` below?

Reply:
This is not used anywhere, this replacement is done automatically. Should I remove them?